GRIB2_Template_4_114_ProductDefinitionTemplate_en.csv

```diff
@@ -0,0 +1,39 @@
+"Average, accumulation, and/or extreme values or other statistically processed values on generalised tiles at a horizontal level or in a horizontal layer in a continuous or non-continuous time interval",56 + (NUTAFTAC-1) - 59 + (NUTAFTAC-1),Scaled Value of Second Fixed Surface,,,,,Operational
+"Average, accumulation, and/or extreme values or other statistically processed values on generalised tiles at a horizontal level or in a horizontal layer in a continuous or non-continuous time interval",60 + (NUTAFTAC-1) - 61 + (NUTAFTAC-1),Year of End of Overall Time Interval,,,,,Operational
+"Average, accumulation, and/or extreme values or other statistically processed values on generalised tiles at a horizontal level or in a horizontal layer in a continuous or non-continuous time interval",62 + (NUTAFTAC-1),Month of End of Overall Time Interval,,,,,Operational
+"Average, accumulation, and/or extreme values or other statistically processed values on generalised tiles at a horizontal level or in a horizontal layer in a continuous or non-continuous time interval",63 + (NUTAFTAC-1),Day of End of Overall Time Interval,,,,,Operational
+"Average, accumulation, and/or extreme values or other statistically processed values on generalised tiles at a horizontal level or in a horizontal layer in a continuous or non-continuous time interval",64 + (NUTAFTAC-1),Hour of End of Overall Time Interval,,,,,Operational
+"Average, accumulation, and/or extreme values or other statistically processed values on generalised tiles at a horizontal level or in a horizontal layer in a continuous or non-continuous time interval",65 + (NUTAFTAC-1),Minute of End of Overall Time Interval,,,,,Operational
+"Average, accumulation, and/or extreme values or other statistically processed values on generalised tiles at a horizontal level or in a horizontal layer in a continuous or non-continuous time interval",66 + (NUTAFTAC-1),Second of End of Overall Time Interval,,,,,Operational
+"Average, accumulation, and/or extreme values or other statistically processed values on generalised tiles at a horizontal level or in a horizontal layer in a continuous or non-continuous time interval",67 + (NUTAFTAC-1),Number of Time Range,,,,,Operational
+"Average, accumulation, and/or extreme values or other statistically processed values on generalised tiles at a horizontal level or in a horizontal layer in a continuous or non-continuous time interval",68 + (NUTAFTAC-1) - 71 + (NUTAFTAC-1),Number of Missing In Statistical Process,,,,,Operational
+"Average, accumulation, and/or extreme values or other statistically processed values on generalised tiles at a horizontal level or in a horizontal layer in a continuous or non-continuous time interval",72 + (NUTAFTAC-1),Type of Statistical Processing,(see code table 4.10),,4.10,,Operational
```

Comment:
@sebvi issue doesn't reference code table, is this CSV more correct?

Reply:
don't understand the comment

Reply:
Ok. did a little more research. This is a code table and the details in the issue summary did not include a reference to it. I concur that the CSV is correct.